### PR TITLE
[#607] Wallet config

### LINKED ISSUE #607: Improve wallet configuration in config file
Now

```yaml
wallet:
  passphrase: pass
address: NfgHwwTi3wHAS8aFAN243C5vGbkYDpqLHP
```

Proposal like in HTTP gateway
```yaml
wallet:
  path: /path/to/wallet.json
  address: NfgHwwTi3wHAS8aFAN243C5vGbkYDpqLHP
  passphrase: pass
```

### PR DESCRIPTION
**Pull request by masterSplinter01**
Close #607

### CODE REVIEW

File: cmd/s3-gw/app_settings.go [hunk 1]
```diff
@@ -32,8 +32,8 @@ const ( // Settings.
 	cfgLoggerLevel = "logger.level"
 
 	// Wallet.
-	cfgWallet           = "wallet"
-	cfgAddress          = "address"
+	cfgWalletPath       = "wallet.path"
```
Comment (KirillovDenis) [inline cmd/s3-gw/app_settings.go:35]: Can we keep flag named `wallet`? And map it to `wallet.path` viper config value?

Comment (alexvanin) [reply cmd/s3-gw/app_settings.go:35]: Here is the example how we bind flag name with viper config value: https://github.com/nspcc-dev/neofs-s3-gw/blob/e5c1acf1e59d7ff9132f7bb5f059687e72ffce59/cmd/s3-gw/app_settings.go#L217-L226

**Comment by KirillovDenis** [comment]
Please update [docs](https://github.com/nspcc-dev/neofs-s3-gw/blob/master/docs/configuration.md#yaml-file-and-environment-variables)

**Comment by masterSplinter01** [comment]
Sorry, I thought I updated docs everywhere

File: docs/configuration.md [hunk 2]
```diff
@@ -175,12 +172,16 @@ default_policy: REP 3
...
+   path: /path/to/wallet.json # Path to wallet
+   passphrase: "" # Passphrase to decrypt wallet.
+   address: NfgHwwTi3wHAS8aFAN243C5vGbkYDpqLHP
 ```
 
 | Parameter    | Type     | Default value | Description                   |
 |--------------|----------|---------------|-------------------------------|
+| `path`       | `string` |               | Path to wallet                |
 | `passphrase` | `string` |               | Passphrase to decrypt wallet. |
+| `address`     | `string`   |               |Account address to get from wallet. If omitted default one will be used.                               |
```
Comment (KirillovDenis) [inline docs/configuration.md:184]: Please align table columns

**Review by KirillovDenis — APPROVED** [review]

**Review by KirillovDenis — APPROVED** [review]

**Pull request merged**